com.github.kagami.boram.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Name=Boram
+GenericName=Video Editor
+GenericName[pt-BR]=Editor de Vídeo
+Comment=Cross-platform graphical WebM converter
```

Comment:
this is inconsistent.

Reply:
Did you mean how `GenericName` says "Video Editor" but `Comment` says "WebM converter"?

I've updated it to say "WebM converter" on both.